### PR TITLE
AHelp Sound Improvements

### PR DESCRIPTION
**Pull request by DEATHB4DEFEAT**
# Description

I don't like the bwoink.
I also don't like being notified about the thing I'm staring at.

If anyone has better quality sounds to suggest, I might use those instead.

---

<details><summary><h1>Media</h1></summary>
<p>

## Different sounds

https://github.com/user-attachments/assets/8003f785-0cd0-44c9-94c2-919c59adac5c

## Ignore sounds when looking

https://github.com/user-attachments/assets/2ac99672-a601-41e0-a5cd-25ce9c1a5c49

</p>
</details>

---

# Changelog

:cl:
- tweak: The AdminHelp sound has changed to three that play under different circumstances

### CODE REVIEW

**Comment by DangerRevolution** [comment]
shouldn't this have an admin CL?

**Comment by DEATHB4DEFEAT** [comment]
> shouldn't this have an admin CL?

No, this affects everyone and we still don't have admin CLs on the bot.

**Review by DangerRevolution — APPROVED** [review]
i like the noise

File: Resources/Audio/Lobby/attributions.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 ﻿- files: ["01 The Gamble.ogg", "02 Guilty Pleasures.ogg", "03 Jazzcuzzi.ogg", "04 The Walk.ogg", "05 Velvet Bossa.ogg", "06 Colors.ogg", "07 Midnight Jam.ogg", "08 Miles Ahead.ogg", "09 Moody.ogg", "10 Flying Away.ogg", "11 Take It Easy.ogg"]
   license: "CC-BY-NC-SA-3.0"
   copyright: "All songs used are produced by Danya Vodovoz, royalty free."
-  source: "https://soundcloud.com/danyavodovoz"
+  source: "https://soundcloud.com/danyavodovoz"
```
Comment (DangerRevolution) [inline Resources/Audio/Lobby/attributions.yml:4]: aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa

Comment (DEATHB4DEFEAT) [reply Resources/Audio/Lobby/attributions.yml:4]:
```suggestion
  source: "https://soundcloud.com/danyavodovoz"

```

**Review by DangerRevolution — APPROVED** [review]
Skibidi noise

**Review by VMSolidus — APPROVED** [review]

**Pull request merged**